doc/design/README.md

```diff
@@ -3,18 +3,17 @@
 In here you can find design documents to all constructs of iceoryx. At the
 moment those design documents are neither complete nor final.
 
-In [draft](./draft/) are design drafts of current features contained which are under
-implementation. The goal is to provide other developers a first idea on how
+Under [draft](./draft/) are design drafts of features stored which are currently under
```

Comment:
I think MkDocs needs a `.md` file to link to and can't link to folders. Maybe remove the link?

```suggestion
Under `/draft` are design drafts of features stored which are currently under
```

Reply:
Again, this is a link which works on github. When mkdocs has a problem with that then we have to adjust mkdocs so that it can handle such things.

Reply:
```suggestion
[Draft](./draft/) stores design drafts of features that are currently under development.
```

Reply:
Stupid question: where can I find the drafts?

Reply:
Sadly github deletes empty folders. It would be located directly here as a subdirectory.